.github/workflows/lint.yaml

```diff
@@ -6,31 +6,42 @@ on:
     paths-ignore:
       - '**/docs/**/*'
 
-
 jobs:
   lint:
-    name: Lint 
+    name: Lint
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
 
-      - name: Use Node.js 
+      - name: Use Node.js
         uses: actions/setup-node@v2
         with:
           node-version: 14
           registry-url: https://registry.npmjs.org/
 
-      - name: Restore cache of node modules.
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+
+      - name: Restore cache of node modules and yarn cache
         uses: actions/cache@v2
         with:
           path: |
             node_modules
             */*/node_modules
-          key: ${{ runner.os }}-lerna-${{ hashFiles('**/yarn.lock') }}
+            ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-nodejs-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-nodejs-
 
       - name: Install dependencies
         run: |
           yarn
+
+      - name: Run shellcheck
+        run: |
+          find . -name "*.sh" ! -ipath "./node_modules/**" | xargs shellcheck -a -Calways
```

Comment:
I'm not sure the regex passed to `-ipath` ignores `packages/node_modules` as well

Reply:
good catch, will change that